### PR TITLE
added markdwon files to single modules

### PR DESCRIPTION
**Pull request by Tobias-Keller**
**Describe the PR**
markdown files can now be added to a single endpoint description.

**Relation issue**
https://github.com/swaggo/swag/issues/506

### CODE REVIEW

File: operation.go [hunk 1]
```diff
@@ -69,6 +69,13 @@ func (operation *Operation) ParseComment(comment string, astFile *ast.File) erro
 	switch lowerAttribute {
 	case "@description":
 		operation.ParseDescriptionComment(lineRemainder)
+	case "@tag.description.markdown":
```
Comment (ubogdan) [inline operation.go]: I think you misunderstood the feature #506. Your code has nothing to do with it.

Also, you need to add "usage" details it to readme.md

**Comment by Tobias-Keller** [comment]
testMarkdown
now it parse a md file for a single endpoint, like in the feature request.
or do I understand the feture request wrong?
I will change the test tomorrow

File: operation.go (hunk 1 above)
Comment (Tobias-Keller) [reply operation.go]: yes, was wrong...please check the my other comment...
https://github.com/swaggo/swag/pull/539#issuecomment-540781023

File: README.md [hunk 2]
```diff
@@ -339,7 +339,7 @@ When a short string in your documentation is insufficient, or you need images, c
 | description.markdown  | A short description of the application. Parsed from the api.md file. This is an alternative to @description    |// @description.markdown No value needed, this parses the description from api.md         																 |
 | tag.name    | Name of a tag.| // @tag.name This is the name of the tag                     |
 | tag.description.markdown   | Description of the tag this is an alternative to tag.description. The description will be read from a file named like tagname.md  | // @tag.description.markdown         |
-
+| endpoint.description.markdown     |  A short description of the application. The description will be read from a file named like endpointname.md| // @description.file endpoint.description.markdown  |
```
Comment (ubogdan) [inline README.md]: As far as understand from the code below, "@endpoint.descrition.markdown" is related to "API operation" not to general API.

**Comment by ubogdan** [comment]
@Tobias-Keller any update here?

**Comment by Tobias-Keller** [comment]
> 
> 
> @Tobias-Keller any update here?

sorry, i had no time in the last days, i make it the days

**Comment by ubogdan** [comment]
@Tobias-Keller we are still waiting for you to finish this PR. 
Take care.

**Comment by Nerzal** [comment]
close this in favor of #630

**Pull request closed without merging**